src/components/Reviews.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+import Review from "./Review";
+import { getReviews } from "../utils/api";
+import "../styles/Reviews.css";
+export default function Reviews() {
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    getReviews().then(({ reviews }) => {
+      console.log("reviews data", reviews);
+      setReviews(reviews);
+    });
+  }, []);
+  return (
+    <ul className="reviews-container">
+      {reviews.length > 0 &&
```

Comment:
Not really sure why you need this check?
If its an issue of displaying something when reviews is empty some sort of loading state might be better?

Reply:
You're right, i'll remove that